package.json

```diff
@@ -20,10 +20,14 @@
     "ci:lint": "npm run lint",
     "ci:build": "npm run build",
     "ci:test": "npm run test",
-    "release": "npm run lint && npm run test",
+    "clean": "rimraf ./dist",
     "version:bump": "gulp version",
+    "conventional-changelog": "conventional-changelog",
     "version:changelog": "npm run conventional-changelog -- -i CHANGELOG.md -s",
-    "conventional-changelog": "conventional-changelog"
+    "publish": "./scripts/publish.sh",
```

Comment:
I think it's better to perform checks here and remove `publish:release` script to avoid possible troubles if running `npm publish`